drink_machines.lua

```diff
@@ -397,6 +417,9 @@ minetest.register_node('drinks:liquid_barrel', {
       if outputstack == 'vessels:glass_bottle' then
          drinks.drinks_liquid_sub(4, 'jbo', 128, pos)
       end
+      if outputstack == 'vessels:steel_bottle' then
```

Comment:
Drinks and drinks two are almost the exact same, but one has hydration values for use with the thirst mod. I'll see if I can't somehow make that still work with only one file.